src/components/Button/types.ts

```diff
@@ -92,4 +92,8 @@ export interface Props extends StyledParentComponent {
      */
     endIcon?: React.ReactNode,
 
+    /**
+     * Whether or not the button is disabled
+     */
+    disabled?: boolean,
```

Comment:
I think this should be isDisabled?

Reply:
I am assuming you are adding `disabled` to have more granular control over the button appearing disabled vs actually being disabled in the `FormSubmit` component, and that causes `Button` to yell at you since it doesn't recognize `disabled`.

If that's the case, I would propose we update `Button` to properly extend the native button element.

Changing line 24 to instead be: 
```
export interface Props extends StyledParentComponent, React.ButtonHTMLAttributes<HTMLButtonElement> {
```
should resolve that without having to manually add `disabled`.